### PR TITLE
Add FreeBSD warning message

### PR DESCRIPTION
**Pull request by felix920506**
Add Warning message for FreeBSD so people know where to ask for support when things go wrong.

### CODE REVIEW

File: docs/general/installation/index.mdx [hunk 1]
```diff
@@ -6,6 +6,14 @@ hide_table_of_contents: true
 
 The Jellyfin project and its contributors offer a number of pre-built binary packages to assist in getting Jellyfin up and running quickly on multiple systems.
 
+:::caution
+[FreeBSD](https://www.freebsd.org/) and its derivatives, such as [TrueNAS CORE](https://www.truenas.com/truenas-core/), are **NOT** supported by Jellyfin and the .NET framework it uses.
```
Comment (Shadowghost) [inline docs/general/installation/index.mdx:10]:
```suggestion
[FreeBSD](https://www.freebsd.org/) and its derivatives, such as [TrueNAS CORE](https://www.truenas.com/truenas-core/), are **NOT** supported by Jellyfin due to .NET officially not being compatible with these platforms.
```

File: docs/general/installation/index.mdx [hunk 2]
```diff
@@ -6,6 +6,14 @@ hide_table_of_contents: true
...
+
+Even though there are builds available online for these platforms, they are unofficial and from [a separate project](https://github.com/Thefrank/jellyfin-server-freebsd). If you do encounter issues on these platforms, please ask for support in their respective support channels FIRST.
```
Comment (Shadowghost) [inline docs/general/installation/index.mdx:12]:
```suggestion
Even though there are builds available online for these platforms, they are unofficial and from [a separate project](https://github.com/Thefrank/jellyfin-server-freebsd). If you do encounter issues on these platforms, please ask for support in their respective support channels first.
```

File: docs/general/installation/index.mdx [hunk 3]
```diff
@@ -6,6 +6,14 @@ hide_table_of_contents: true
...
+
+[TrueNAS SCALE](https://www.truenas.com/truenas-scale/) is based on Linux and therefore is officially supported. Please install the Jellyfin app from its app repository.
```
Comment (Shadowghost) [inline docs/general/installation/index.mdx:14]:
```suggestion
[TrueNAS SCALE](https://www.truenas.com/truenas-scale/) is based on Linux and therefore officially supported. Please install the Jellyfin app from its app repository. This app is not officially maintained by the Jellyfin team, therefore please use the TrueNAS support channels for help first.
```

**Comment by felix920506** [comment]
Hi. I think all suggestions look good and I have accepted all of them.

**Review by Shadowghost — APPROVED** [review]

**Pull request merged**